### PR TITLE
init bridge nat

### PR DESCRIPTION
**Pull request by qingwusunny**
init bridge nat flow

### CODE REVIEW

**Review by KeyongSun — DISMISSED** [review]

**Comment by wu-cl** [comment]
modify commit messages, or CI will fail after merge

**Comment by qingwusunny** [comment]
> modify commit messages, or CI will fail after merge
done

File: pkg/agent/datapath/id_generate.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+func (i *IDGenerate) ascendUint64() {
+	i.lock.Lock()
+	defer i.lock.Unlock()
+
+	i.idUint64++
+}
+
+var LearnCookieID = &IDGenerate{}
```
Comment (zwtop) [inline pkg/agent/datapath/id_generate.go:33]: It is ok to let `LearnCookieID` and `IDGenerate` be unexported?

File: pkg/agent/datapath/utils_test.go [hunk 2]
```diff
@@ -103,3 +104,36 @@ func TestMatchPort(t *testing.T) {
...
+			src:  uint64(0xff16),
+			res:  []byte{0, 0, 0, 0, 0, 0, 255, 22},
+		}, {
+			name: "invalid type",
+			src:  uint8(2),
+			res:  nil,
+		},
+	}
+
+	for _, test := range tests {
```
Comment (zwtop) [inline pkg/agent/datapath/utils_test.go:133]: nit: put each case into `t.Run()`

File: pkg/agent/datapath/natBridge.go [hunk 3]
```diff
@@ -0,0 +1,515 @@
...
+	chooseBackendFlagField := ofctrl.LearnField{Name: ChooseBackendFlagReg, Start: uint16(ChooseBackendFlageStart)}
+
+	cookieID, err := getLearnCookieID()
+	if err != nil {
+		return nil, err
+	}
+	learnAct := ofctrl.NewLearnAction(NatBrSessionAffinityTable, MID_MATCH_FLOW_PRIORITY, 0, 300, 0, 0, cookieID)
+	learnAct.SetDeleteLearned()
+
+	if err := learnAct.AddLearnedMatch(&ethTypeField, 16, nil, uintToByteBigEndian(uint16(PROTOCOL_IP))); err != nil {
```
Comment (zwtop) [inline pkg/agent/datapath/natBridge.go:389]: nit: replace the magic number with a named constant

File: pkg/agent/datapath/utils_test.go (hunk 2 above)
Comment (qingwusunny) [reply pkg/agent/datapath/utils_test.go:133]: done

File: pkg/agent/datapath/natBridge.go (hunk 3 above)
Comment (qingwusunny) [reply pkg/agent/datapath/natBridge.go]: done

File: pkg/agent/datapath/id_generate.go (hunk 1 above)
Comment (qingwusunny) [reply pkg/agent/datapath/id_generate.go]: done

**Review by zwtop — APPROVED** [review]

**Review by KeyongSun — APPROVED** [review]

**Pull request merged**